src/test/regress/expected/multi_transaction_recovery.out

```diff
@@ -41,9 +43,9 @@ CREATE TABLE should_be_sorted_into_middle (value int);
 PREPARE TRANSACTION 'citus_0_should_be_sorted_into_middle';
 SET citus.force_max_query_parallelization TO ON;
 -- Add "fake" pg_dist_transaction records and run recovery
-INSERT INTO pg_dist_transaction VALUES (1, 'citus_0_should_commit'),
+INSERT INTO pg_dist_transaction VALUES (:worker_group_id, 'citus_0_should_commit'),
                                        (0, 'citus_0_should_commit');
-INSERT INTO pg_dist_transaction VALUES (1, 'citus_0_should_be_forgotten'),
+INSERT INTO pg_dist_transaction VALUES (:worker_group_id, 'citus_0_should_be_forgotten'),
```

Comment:
the test file change is triggering our "flaky test detection", which enforces that all test files are repeatable and not-flaky by repeating the tests that are touched by the PR many times. We did not enforce repeatability on existing tests, but any change triggers enforcement.

if you prefer, you can revert this change to avoid triggering it (we'll deal with it later), or try to change the test file to be repeatable

Reply:
@marcocitus, done